## Compressed Gas Volume

Calculate the amount of the gas (in terms of volume roughly at 1 atm and room temperature) in a compressed gas bottle
- The purpose is to estimate how many gas bottles I have to order for the LAr filter regeneration, for what the amount of Ar and Ar+2%H2 gas is estimated from the manufacturer

- Gas bottle: ~2500 psi and room temperature
    - Supercritical fluids; not gas nor liquid
    - Reference: https://www.researchgate.net/figure/Argon-pressure-temperature-phase-diagram-with-the-new-thermodynamic-boundary-The-phase_fig2_272295671
    - It can also be found in any argon phase diagram

- Argon data from NIST: https://webbook.nist.gov/chemistry/fluid/
    - Isothermal properties
    - Default for fluid
    - 300 K, 2250 psia: 261.88 kg/m3
    - 300 K, 2400 psia: 279.24 kg/m3 (5 times less dense than LAr)
    - Consistent with Argon property source 1 below

- Argon property source 1, http://cmwelding.com/technical/gas-calculator-gas-phase
    - 70 F, 2500 psi, water volume (the volume of the bottle) 1.7 cu.ft.
    - Density: 18.76047 lbm/ft3 (=0.3g/cm3)
    - Compressibility: 181.522
    - Product volume: 308.59 scf
    - Product weight: 31.89 lbs
    - The density is 4.67 times smaller than LAr
    - This is very close to the ideal gas, in which case the compressibility is 2500/14.5 = 172.4
    

- Argon property source 2, the engineering book, https://www.engineeringtoolbox.com/argon-density-specific-weight-temperature-pressure-d_2089.html
    - 300 K, 1450 psia
    - Density: 167.6 kg/m3 (=0.1676g/cm3)
    - the density is 8 times smaller than LAr

- Gas argon density at 1 bara and 25 C: 1.613 kg/m3
- Liquid argon density at 1 bara and 87.18 K: 1396 kg/m3

- Information from Airgas
    - UHP Ar in Size 300 High Pressure Steel Cylinder, CGA-580, pressure 2400 psi, internal volume 49.8 L, 336 ft3 (9514.46 L), environmental temperature
    - 2% H2+Ar in Size 200 High Pressure Steel Cylinder, CGA-350, pressure 2250 psi, internal volume 43.9 L, 251 ft3 (7107.528), environmental temperature
    - They use the temperature of 70 F to calculate 

- Expanding factor from liquid to gas

In [1]:
f = 1396/1.613
f

865.4680719156851

- Expanding factor of argon property source 1

In [2]:
f/4.67

185.32506893269488

- Use the ideal gas law in the case of argon property source 1, compare the pressure ratio and see the expanding factor

In [3]:
2500/14.5

172.41379310344828

- Conclusion: The expanding factor (185) is very close to that calculated from the ideal gas law (172)

- NIST database: compare the density of LAr and Ar at 2400 psia, 300K

In [4]:
1396/279.24

4.9992837702334905

- There is a factor of 5 difference; i.e. LAr is 5 times denser than Ar at 2400 psia, 300K.
- We assumed Ar in the Ar gas bottle is LAr last time.  It results in 5 times overestimation of the Ar gas amount.

- Compare the density of the gas Ar at the room temperature, pressure, with that at 2400 psia, 300K in the Size 300 High Pressure Steel Cylinder
- Calculate the Ar gas volume at the room temperature, pressure for the Ar in the cylinder

In [5]:
Cf300 = 279.24/1.613
Vol300 = 49.8*Cf300
print( Cf300, Vol300 )

173.1184128952263 8621.296962182269


- Similarly, compare the density of the gas Ar at the room temperature, pressure, with that at 2250 psia, 300K in the Size 200 High Pressure Steel Cylinder
- Calculate the Ar gas volume at the room temperature, pressure for the Ar in the cylinder

In [6]:
Cf200 = 261.88/1.613
Vol200 = 43.9*Cf200
print( Cf200, Vol200 )

162.3558586484811 7127.42219466832


- Use the Ar gas volume from Airgas (9514.46L for UHP Ar), calculate how long it will be finished at 160 lpm (in minutes)

In [7]:
9514.46/160

59.465374999999995

- Use the Ar gas volume from Airgas (7107.528L for 2%H<sub>2</sub>+Ar mixure), calculate how long it will be finished at 160 lpm (in minutes)

In [8]:
7107.528/160

44.42205

- Instruction from Research Catalysts for preheating:
    - At least 5 SLPM per kg of catalyst is recommended for efficient heating of the catalyst.
    - Preheating will last for at least two hours
    - Total Ar needed to preheat 10 kg (molecular+copper)
- Assume 10 kg of sieves (molecular+copper)

In [11]:
TotalAr = 5*10*60*2
print('Need %d L of Ar gas for preheating' % TotalAr)

Need 6000 L of Ar gas for preheating


- Bern uses 160 slpm for 20 kg of sieves for about 2 hours

In [12]:
TotalArBern = 160*60*2
print('Total Ar used in Bern: %d L for preheating' % TotalArBern )

Total Ar used in Bern: 19200 L for preheating


- Instruction from Research Catalysts for regenerating the copper sieves:
    - 5 slpm per kg, it takes ~24 hours
    - 15 slpm per kg, it takes ~5 hours
- Assume 5 kg of copper sieves

In [14]:
TotalMix1 = 5*5*60*24
TotalMix2 = 15*5*60*5
print('Need %d at 5 slpm per kg or %d at 15 slpm per kg for regenerating 5 kg of copper sieves' % (TotalMix1, TotalMix2) )

Need 36000 at 5 slpm per kg or 22500 at 15 slpm per kg for regenerating 5 kg of copper sieves


- Conclusion:
    - 5-6 bottles Ar
    - 6-7 bottles of Mix